app/src/main/AndroidManifest.xml

```diff
@@ -33,83 +32,85 @@
+                <category android:name="android.intent.category.BROWSABLE" />
 
-                <data android:scheme="http"/>
-                <data android:scheme="https"/>
-                <data android:host="androidmakers.fr"/>
-                <data android:path="/"/>
+                <data android:scheme="http" />
+                <data android:scheme="https" />
+                <data android:host="androidmakers.fr" />
+                <data android:path="/" />
             </intent-filter>
 
             <meta-data
                 android:name="default-url"
-                android:value="https://androidmakers.fr/"/>
+                android:value="https://androidmakers.fr/" />
 
             <intent-filter android:autoVerify="true">
-                <action android:name="android.intent.action.VIEW"/>
+                <action android:name="android.intent.action.VIEW" />
 
-                <category android:name="android.intent.category.DEFAULT"/>
-                <category android:name="android.intent.category.BROWSABLE"/>
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
 
-                <data android:scheme="http"/>
-                <data android:scheme="https"/>
-                <data android:host="androidmakers.fr"/>
-                <data android:pathPrefix="/schedule"/>
-                <data android:pathPrefix="/logistics"/>
+                <data android:scheme="http" />
+                <data android:scheme="https" />
+                <data android:host="androidmakers.fr" />
+                <data android:pathPrefix="/schedule" />
+                <data android:pathPrefix="/logistics" />
             </intent-filter>
         </activity>
 
-        <activity android:name=".ui.activity.AboutActivity"/>
+        <activity android:name=".ui.activity.AboutActivity" />
         <activity
             android:name=".ui.activity.SessionDetailActivity"
-            android:configChanges="orientation|screenSize"/>
+            android:configChanges="orientation|screenSize" />
 
         <receiver
             android:name=".receiver.SessionAlarmReceiver"
             android:exported="false">
             <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED"/>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
             </intent-filter>
         </receiver>
 
         <service
             android:name=".service.SessionAlarmService"
             android:exported="false"
-            android:permission="android.permission.BIND_JOB_SERVICE"/>
+            android:permission="android.permission.BIND_JOB_SERVICE" />
 
         <service
-            android:name=".service.MyFirebaseMessagingService">
+            android:name=".service.MyFirebaseMessagingService"
+            android:exported="true">
```

Comment:
No need to export this service if I remember well :)

Reply:
I had a error (or maybe warning) because there's an `intent-filter` and so it should be exported 🤷 . Overall, I haven't tested notifications at all so that's something on the todo list...

Reply:
Yes the service has to have the property exported but set to false :) 
```suggestion
            android:exported="false">
```